randomness-agent/src/lib.rs

```diff
@@ -13,7 +13,7 @@ use std::future::Future;
 use std::sync::Arc;
 
 /// Scheme ID of the randomness scheme
-pub const RANDOMNESS_SCHEME_ID: &str = "BN254";
+pub const RANDOMNESS_SCHEME_ID: &str = "BLS12381";
```

Comment:
Does this really need to be in the lib and imported later? It looks like now that we have "configurable" schemes, it would make more sense to have it just be a parameter that lives in the examples like the choice of the signature scheme, no?

cc @azixus